moveit_experimental/moveit_jog_arm/src/low_pass_filter.cpp

```diff
@@ -63,12 +92,12 @@ double LowPassFilter::filter(double new_measurement)
   previous_measurements_[1] = previous_measurements_[0];
   previous_measurements_[0] = new_measurement;
 
-  double new_filtered_msrmt = (1. / (1. + filter_coeff_)) * (previous_measurements_[1] + previous_measurements_[0] -
-                                                             (-filter_coeff_ + 1.) * previous_filtered_measurement_);
+  double new_filtered_measurement = scale_term_ * (previous_measurements_[1] + previous_measurements_[0] -
```

Comment:
Why add those extra variables? `filter_coeff_` is redundant now, the warnings would also work for `low_pass_filter_coeff`. With the original implementation the terms would be optimized anyway.

Reply:
@tylerjw no anwer?